bottles/frontend/ui/about.blp

```diff
@@ -4,6 +4,7 @@ using Gtk 4.0;
   application-name: "Bottles";
   modal: true;
   version: "2022.9.28.1";
+  comments: "Run Windows Software";
```

Comment:
This is discouraged by GNOME developers. https://matrix.to/#/!hwiGbsdSTZIwSRfybq:matrix.org/$16602639644946QoBIj:gnome.org?via=matrix.org&via=gnome.org&via=privacytools.io

![](https://user-images.githubusercontent.com/50847364/194077155-99080b33-f351-42da-815b-5ba9e26103db.png)
```suggestion
```

Reply:
ok i removed it do you have any more things you think i should do with the pr?

Reply:
Now we just need to test it and see if the change causes issues or not.